### PR TITLE
Made all models swappable

### PR DESCRIPTION
**Pull request by thedrow**
This is useful for having an access model token that encrypts the tokens in the database using pgcrypto fields or for replacing the user fields with denormalised user fields using [django-denorm](https://github.com/django-denorm/django-denorm).
There are a lot of other cases that require all models to be swappable. We can discuss them if you want.
Unfortunately I uncovered a regression in related fields of swappable models at least in Django 1.9 on Python 2.7.
We need to figure out why and fix it if possible.

### CODE REVIEW

**Comment by thedrow** [comment]
The regression is present in Django 1.9 on all Python versions.
See https://travis-ci.org/evonove/django-oauth-toolkit/jobs/108660974#L171

**Comment by thedrow** [comment]
Any idea why the tests fail on other Django versions as well?

**Comment by thedrow** [comment]
And now you can see the failure in 1.7 as well. https://travis-ci.org/evonove/django-oauth-toolkit/jobs/108665464#L171

File: oauth2_provider/admin.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
 from django.contrib import admin
 
-from .models import Grant, AccessToken, RefreshToken, get_application_model
+from .models import (get_grant_model, get_access_token_model,
+                     get_refersh_token_model, get_application_model)
```
Comment (agriffis) [inline oauth2_provider/admin.py]: `s/get_refersh_token_model/get_refresh_token_model/g`

**Comment by menecio** [comment]
Any chance to help with this one?

**Comment by thedrow** [comment]
You can take it from here. I'm no longer interested in working on this.

**Comment by menecio** [comment]
Implemented on https://github.com/evonove/django-oauth-toolkit/pull/467 and landed on master. 

@jleclanche I think this one can be closed now.

**Pull request closed without merging**